### PR TITLE
August 30 update

### PR DESCRIPTION
**Pull request by mlutze**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,3 +104,41 @@ you may instead play a collection of cards such that:
...
+_Added by Mathias, 30 Aug 2023_
+
+If you draw a card on your turn instead of playing another move,
+you may play the drawn card if it is otherwise valid to do so.
+
+## Honorable Penalty
+_Added by Xavier, 30 Aug 2023_
+
+If you are to draw a penalty card,
+you must first bow to the Dealer.
```
Comment (xdesou01) [inline README.md]: eye contact clause?

File: README.md [hunk 2]
```diff
@@ -104,3 +104,41 @@ you may instead play a collection of cards such that:
...
+_Added by Jonathan, 30 Aug 2023_
+
+The Dealer is designated by the Dealer Token.
+If you are Dealer, you may not touch the cards unless you are in possession of the Token.
+
+## Official Cut
+_Added by Matt, 30 Aug 2023_
+
+If you are Dealer, after shuffling, you must designate a Cutter.
+If you are Cutter, you must then receive the Dealer Token, then the deck,
```
Comment (JonathanStarup) [inline README.md]: "[...], you must then receive the deck and the dealer token before proceeding, [...]"

This allows for trickery

File: README.md [hunk 3]
```diff
@@ -104,3 +104,41 @@ you may instead play a collection of cards such that:
...
+cut the deck, and return the Token and deck to the Dealer.
+If you are Dealer, you may not thereafter manipulate the cards except to deal them.
+
+## Slam
+_Added by Mathias, 30 Aug 2023_
+
+If you draw a card on your turn instead of playing another move,
+you may play the drawn card if it is otherwise valid to do so.
```
Comment (JonathanStarup) [inline README.md]: In Mathias' phrasing it was more general

On your turn you are allowed to draw a card and then play a card (or a valid set). The turn is assumed to end on a card draw so play fast before the next player takes the turn.

File: README.md (hunk 1 above)
Comment (JonathanStarup) [reply README.md]: [...] you must first have eye contact with the dealer and bow.

File: README.md (hunk 2 above)
Comment (mlutze) [reply README.md]: I've modified the phrasing. Does this fix it?

File: README.md [hunk 4]
```diff
@@ -115,21 +115,22 @@ If you are Dealer, you may not touch the cards unless you are in possession of t
...
+If you are Cutter, you must then receive the Dealer Token and the deck,
+then cut the deck, and return the Token and deck to the Dealer.
 If you are Dealer, you may not thereafter manipulate the cards except to deal them.
 
 ## Slam
 _Added by Mathias, 30 Aug 2023_
 
 If you draw a card on your turn instead of playing another move,
-you may play the drawn card if it is otherwise valid to do so.
+you may play the drawn card or a collection of cards including the drawn card
```
Comment (JonathanStarup) [inline README.md]: ~including the drawn card~

**Review by JonathanStarup — APPROVED** [review]

**Pull request merged**